serverless/event_workflows/serverless-channels.adoc

```diff
@@ -12,18 +12,24 @@ image::serverless-event-channel-workflow.png[Channel workflow overview]
 
 After events have been sent to a channel from an event source or producer, these events can be sent to multiple Knative services, or other sinks, by using a subscription.
 
-include::modules/serverless-supported-channel-types.adoc[leveloffset=+1]
-// include::modules/serverless-inmemorychannel.adoc[leveloffset=+2]
-// comment out for now - this information isn't necessary while only default channel type is available, and should be moved to an admin guide anyway.
+{ServerlessProductName} supports `InMemoryChannel` and `KafkaChannel` channels for development use.
 
-[id="serverless-channels-creating"]
-== Creating channels
+:FeatureName: Apache Kafka on {ServerlessProductName}
+include::modules/technology-preview.adoc[leveloffset=+2]
 
-Developers can create channels by instantiating a supported `Channel` object.
+The following are limitations of `InMemoryChannel` type channels:
+
+* No event persistence is available. If a pod goes down, events on that pod are lost.
+* InMemoryChannel type channels do not implement event ordering, so two events that are received in the channel at the same time can be delivered to a subscriber in any order.
+* If a subscriber rejects an event, there are no re-delivery attempts by default. You can configure re-delivery attempts by adding a dead letter sink to the `delivery` spec in the `Subscription` object.
```

Comment:
re-delivery (`retry` , together with `backoffPolicy` and `backoffDelay` )  and dead letter sink (`deadLetterSink` ) are two separate things that can be configured in `spec.delivery`   ( you can potentially have retries without a dead letter sink and vice versa)

Reply:
Let's open up a follow up Jira maybe to cover re-delivery docs and dead letter sink properly - I know we added some stuff like this about dead letter sink in the past, but I think it warrants having a better explanation in subscriptions docs maybe about configuring re-delivery in the recommended ways.